dropwizard-metrics/dropwizard-metrics5/src/main/java/com/wavefront/integrations/metrics/WavefrontReporter.java

```diff
@@ -112,29 +134,6 @@ public Builder withSource(String source) {
       return this;
     }
 
-    /**
-     * Set the Point Tags for this reporter.
-     *
-     * @param pointTags the pointTags Map for all metrics
-     * @return {@code this}
-     */
-    public Builder withPointTags(Map<String, String> pointTags) {
```

Comment:
My earlier comment was to actually keep this (global reporter level tags) and combine them with the per metric tags at report time. This likely will increase allocations during reporting.

The ideal solution would have been for dropwizard to support global tags at the registry level vs having to handle it on the reporter side. But users have the option of including any global tags when registering the metrics.